acceptor/private.local

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
Please rename the file to smth like `local_settings.py.example`
It will be easy to rename it and its name explains the purpose of the file